tests/tf-manifests/aws/account-roles/rosa-classic/main.tf

```diff
@@ -15,9 +15,18 @@ provider "rhcs" {
   url = var.url
 }
 
+data "aws_caller_identity" "current" {
+}
+
+data "aws_partition" "current" {
+}
+
 locals {
+  path = coalesce(var.path, "/")
   major_version = "${split(".", var.openshift_version)[0]}.${split(".", var.openshift_version)[1]}"
   versionfilter = var.openshift_version == null ? "" : " and raw_id like '%${local.major_version}%'"
+  installer_role_arn = "arn:${data.aws_partition.current.partition}:iam::${data.aws_caller_identity.current.account_id}:role${local.path}${var.account_role_prefix}-Installer-Role"
```

Comment:
Where is that one used ?

Reply:
it's used by the output of account roles https://github.com/terraform-redhat/terraform-provider-rhcs/pull/569/files#diff-af47971dcdc8b7bd6bff3593ed96629cb6ce60b778725013948b3e9c5e133d0aR429, and `PrepareSharedVpcPolicyAndHostedZone` will consume it.

Reply:
ok